pkg/constants/constants.go

```diff
@@ -418,12 +419,13 @@ const (
 	VSphereSecretNameKey      = "vsphere_secret_name"
 	DefaultSecretName         = "velero-vsphere-config-secret"
 	DefaultSecretNamespace    = "velero"
+	UploadCRRetryMaximumKey   = "upload-cr-retry-max"
 )
 
 const (
 	// AnnVolumeHealth is the key for HealthStatus annotation on volume claim
 	// for vSphere CSI Driver.
 	AnnVolumeHealth = "volumehealth.storage.kubernetes.io/health"
-        // key for expressing timestamp for volume health annotation
-        AnnVolumeHealthTS = "volumehealth.storage.kubernetes.io/health-timestamp"
+	// key for expressing timestamp for volume health annotation
+	AnnVolumeHealthTS = "volumehealth.storage.kubernetes.io/health-timestamp"
```

Comment:
@deepakkinni Any idea why we need these in our constants?  These are vSphere CSI driver annotations.

Reply:
https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/0f02ba2461e03f09773606c3cf316d509d3a6941/pkg/plugin/backup_pvc_action_plugin.go#L207-L211